packages/inngest/src/components/Inngest.ts

```diff
@@ -477,6 +510,36 @@ export class Inngest<TClientOpts extends ClientOptions = ClientOptions>
     );
   }
 
+  // biome-ignore lint/correctness/noUnusedPrivateClassMembers: used in the SDK
+  private async _updateMetadata({
+    target,
+    metadata,
+    headers,
+  }: {
+    target: MetadataTarget;
+    metadata: Array<{
+      kind: string;
+      op: string;
+      values: Record<string, unknown>;
+    }>;
+    headers?: Record<string, string>;
+  }): Promise<void> {
+    const res = await this.inngestApi.updateMetadata(
+      { target, metadata },
+      { headers: { ...this.headers, ...headers } },
+    );
+    if (res.ok) {
+      console.log("We got this response: ", res);
+      return res.value;
+    }
+
+    console.log("Failed to update metadata!");
```

Comment:
Dangling logs. We could use `client.logger.*()` if we do still need to add something.